### PR TITLE
Flip card animation

### PR DESCRIPTION
**Pull request by filwas**

### CODE REVIEW

File: src/components/Card.module.css [hunk 1]
```diff
@@ -50,3 +49,20 @@
...
+@keyframes flip-vertical-right {
+  0% {
+    transform: rotateY(0);
+  }
+  50% {
+    transform: rotateY(90deg);
+  }
+  100% {
+    transform: rotateY(0);
+  }
```
Comment (filwas) [inline src/components/Card.module.css:66]: I'm really fond of this "hack". Initially I was rotating by 180 degrees, which caused everything to be a mirror-image, and the text was changing too soon. So, I added a timeout that makes the text change happen exactly in the middle - when the card is just a straight line - and made the animation only turn by 90 degrees, just to make it go back. Visually it tricks the mind that a flip is happening, but in reality it was really simple to implement.

This task reminded me of the mouse cursor animation during loading in the old game Battlefield 2. I cannot find a video link, but it was basically the same trick as the one I used here.

File: src/components/Card.tsx [hunk 2]
```diff
@@ -83,6 +93,11 @@ export const Card = (props: CardProps) => {
     setTextOutputHeight(target.scrollHeight);
   };
 
+  const dynamicClasses = [
+    styles.card,
+    playAnimation ? styles.flipVerticalRight : "",
+  ].join(" ")
```
Comment (filwas) [inline src/components/Card.tsx:96]: I couldn't find it in the design, but initially I was thinking of making two animations - right flip and left flip - which would play depending on which side was displayed initially. If you think I should add it, I will.

**Review by Sumick — CHANGES_REQUESTED** [review]
Rejecting, as you have a lot of things to fix from previous PRs

**Review by Sumick — CHANGES_REQUESTED** [review]
There is a lot of old stuff here - let's finish that first PR, merge main into it so it's clear what is happening in terms of this specific task requirements

**Review by Sumick — APPROVED** [review]

**Pull request merged**